website/source/docs/telemetry/metrics.html.md

```diff
@@ -0,0 +1,805 @@
+    <td>Percentage</td>
+    <td>Gauge</td>
+    <td>node_id, datacenter, cpu</td>
+  </tr>
+  <tr>
+    <td>`nomad.client.host.cpu.user`</td>
+    <td>CPU utilization in the user space</td>
+    <td>Percentage</td>
+    <td>Gauge</td>
+    <td>node_id, datacenter, cpu</td>
+  </tr>
+  <tr>
+    <td>`nomad.client.host.cpu.system`</td>
+    <td>CPU utilization in the system space</td>
+    <td>Percentage</td>
+    <td>Gauge</td>
+    <td>node_id, datacenter, cpu</td>
+  </tr>
+  <tr>
+    <td>`nomad.client.host.cpu.idle`</td>
+    <td>Idle time spent by the CPU</td>
+    <td>Percentage</td>
+    <td>Gauge</td>
+    <td>node_id, datacenter, cpu</td>
+  </tr>
+  <tr>
+    <td>`nomad.client.host.disk.size`</td>
+    <td>Total size of the device</td>
+    <td>Bytes</td>
+    <td>Gauge</td>
+    <td>node_id, datacenter, disk</td>
+  </tr>
+  <tr>
+    <td>`nomad.client.host.disk.used`</td>
+    <td>Amount of space which has been used</td>
+    <td>Bytes</td>
+    <td>Gauge</td>
+    <td>node_id, datacenter, disk</td>
+  </tr>
+  <tr>
+    <td>`nomad.client.host.disk.available`</td>
+    <td>Amount of space which is available</td>
+    <td>Bytes</td>
+    <td>Gauge</td>
+    <td>node_id, datacenter, disk</td>
+  </tr>
+  <tr>
+    <td>`nomad.client.host.disk.used_percent`</td>
+    <td>Percentage of disk space used</td>
+    <td>Percentage</td>
+    <td>Gauge</td>
+    <td>node_id, datacenter, disk</td>
+  </tr>
+  <tr>
+    <td>`nomad.client.host.disk.inodes_percent`</td>
+    <td>Disk space consumed by the inodes</td>
+    <td>Percent</td>
+    <td>Gauge</td>
+    <td>node_id, datacenter, disk</td>
+  </tr>
+  <tr>
+    <td>`nomad.client.allocs.start`</td>
+    <td>Number of allocations starting</td>
+    <td>Integer</td>
+    <td>Counter</td>
+    <td>node_id, job, task_group</td>
+  </tr>
+  <tr>
+    <td>`nomad.client.allocs.running`</td>
+    <td>Number of allocations starting to run</td>
+    <td>Integer</td>
+    <td>Counter</td>
+    <td>node_id, job, task_group</td>
+  </tr>
+  <tr>
+    <td>`nomad.client.allocs.failed`</td>
+    <td>Number of allocations failing</td>
+    <td>Integer</td>
+    <td>Counter</td>
+    <td>node_id, job, task_group</td>
+  </tr>
+  <tr>
+    <td>`nomad.client.allocs.restart`</td>
+    <td>Number of allocations restarting</td>
+    <td>Integer</td>
+    <td>Counter</td>
+    <td>node_id, job, task_group</td>
+  </tr>
+  <tr>
+    <td>`nomad.client.allocs.complete`</td>
+    <td>Number of allocations completing</td>
+    <td>Integer</td>
+    <td>Counter</td>
+    <td>node_id, job, task_group</td>
+  </tr>
+  <tr>
+    <td>`nomad.client.allocs.destroy`</td>
+    <td>Number of allocations being destroyed</td>
+    <td>Integer</td>
+    <td>Counter</td>
+    <td>node_id, job, task_group</td>
+  </tr>
+</table>
+
+Nomad 0.9 adds an additional "node_class" label from the client's
+`NodeClass` attribute. This label is set to the string "none" if empty.
+
+## Host Metrics (deprecated post Nomad 0.7)
```

Comment:
@preetapan : do we want to remove these from the documentation, in preparation for removing them from the codebase?